package.json

```diff
@@ -45,6 +45,7 @@
     "react-dom": "16.9.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz",
     "react-native-autocomplete-input": "4.2.0",
+    "react-native-check-box": "2.1.7",
```

Comment:
probably will remove this now since you're using react-native-paper?

Reply:
yes